Unfortunately, it seems upstream hasn't added all the needed libraries for linking... fix that

--- kdebindings/ruby/qtscript/CMakeLists.txt
+++ kdebindings/ruby/qtscript/CMakeLists.txt
@@ -4,7 +4,7 @@
 set(qtscript_LIBRARY_SRC qtscript.cpp qtscripthandlers.cpp)
 
 add_library(qtscript MODULE ${qtscript_LIBRARY_SRC})
-target_link_libraries(qtscript ${QT_QTCORE_LIBRARY} ${RUBY_LIBRARY} smokeqtcore smokeqtscript qtruby4shared)
+target_link_libraries(qtscript ${QT_QTCORE_LIBRARY} ${QT_QTSCRIPT_LIBRARY} ${RUBY_LIBRARY} smokeqtcore smokeqtscript qtruby4shared)
 set_target_properties(qtscript PROPERTIES PREFIX "")
 install(TARGETS qtscript DESTINATION ${CUSTOM_RUBY_SITE_ARCH_DIR})
 install(FILES qtscript.rb DESTINATION ${CUSTOM_RUBY_SITE_LIB_DIR}/qtscript)
--- kdebindings/ruby/soprano/CMakeLists.txt
+++ kdebindings/ruby/soprano/CMakeLists.txt
@@ -3,7 +3,7 @@
 
 set(rubysoprano_LIBRARY_SRC soprano.cpp sopranohandlers.cpp)
 add_library(rubysoprano MODULE ${rubysoprano_LIBRARY_SRC})
-target_link_libraries(rubysoprano ${QT_QTCORE_LIBRARY} ${RUBY_LIBRARY} smokeqtcore smokesoprano qtruby4shared)
+target_link_libraries(rubysoprano ${QT_QTCORE_LIBRARY} ${RUBY_LIBRARY} ${SOPRANO_LIBRARIES} smokeqtcore smokesoprano qtruby4shared)
 set_target_properties(rubysoprano PROPERTIES PREFIX "" OUTPUT_NAME soprano)
 install(TARGETS rubysoprano DESTINATION ${CUSTOM_RUBY_SITE_ARCH_DIR})
 install(FILES soprano.rb DESTINATION ${CUSTOM_RUBY_SITE_LIB_DIR}/soprano)
--- kdebindings/ruby/nepomuk/CMakeLists.txt
+++ kdebindings/ruby/nepomuk/CMakeLists.txt
@@ -7,6 +7,7 @@
 target_link_libraries(rubynepomuk 
     ${RUBY_LIBRARY} 
     ${NEPOMUK_LIBRARIES}
+    ${SOPRANO_LIBRARIES}
     smokeqtcore 
     smokesoprano
     smokekdecore 
